wafer/settings.py

```diff
@@ -325,3 +325,8 @@
     'wafer.users.views.UsersView',
     'wafer.users.views.ProfileView',
 )
+
+try:
+    from wafer.localsettings import *
```

Comment:
How about doing this the other way around. So that it only impacts dev, not library consumers?

`manage.py --shell localsettings`

Reply:
Would that work? AFAICS manage.py will set the `DJANGO_SETTINGS_MODULE` environment variable to  the settings.py file regardless.

Maybe we need documentation section explaining how to setup a local settings file and best use it?

Reply:
nah, I guess anyone hacking on wafer needs to be aware of how Django works anyway. maybe we should just drop this